api/hub/src/datasources/moderation-decision.ts

```diff
@@ -250,14 +261,17 @@ class ModerationDecisionAPI extends DataSource {
     if (await queryCache.has(decisionCache)) {
       return queryCache.get(decisionCache);
     }
-    const decision = await this.getDecision(contentID);
+    let finalDecision = await this.getDecision(contentID);
     // remove action log as it is not needed here
-    let { finalDecision } = decision;
+    if (finalDecision.actions) {
+      delete finalDecision.actions;
+    }
     // add moderator data
-    if (finalDecision.moderator) {
+    if (finalDecision.moderator && finalDecision.moderator.length) {
       const moderator = finalDecision.moderator.startsWith('0x')
         ? await profileAPI.getProfile(finalDecision.moderator)
         : await profileAPI.resolveProfile(finalDecision.moderator);
+      finalDecision.moderator = moderator.pubKey;
       finalDecision = Object.assign({}, finalDecision, {
```

Comment:
this can be just a simple Object.assign with the first paramenter being `finalDecision`

Reply:
Here I'm overwriting the `moderator` attribute to make sure it will always be the pubKey. I'm not creating a new attribute.

Reply:
so why not use a normal assignment? :D

Reply:
btw `Object.assign(finalDecision, {....})` does exactly what you want to achieve with the reassignment

Reply:
Oh, I see. OK, I've switched to a normal assignment.